docs/testing.md

```diff
@@ -0,0 +1,18 @@
+# Testing
+
+## Backend
+
+Backend testing includes both unit testing and integration testing. Because manual testing is less efficient, all new features or code changes need defence from automated testing. Without it, who knows when things will break.
```

Comment:
```suggestion
Backend testing includes both unit testing and integration testing. Because manual testing is less efficient, all new features or code changes need defence from automated testing. Without it, no one knows when things will break.
```